<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestTableWidget</class>
 <widget class="QMainWindow" name="TestTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>414</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TestTableWidget</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitterH">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="GB_button">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="8" column="0">
         <widget class="QCheckBox" name="CB_03horizontalVisible">
          <property name="text">
           <string>显示水平表头</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="PB_04insertRow">
          <property name="text">
           <string>插入行</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="PB_03initalizeTable">
          <property name="text">
           <string>初始化表格数据</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="PB_08autoColumnWidth">
          <property name="text">
           <string>自动调节列宽</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QCheckBox" name="CB_01tableEditable">
          <property name="text">
           <string>表格可编辑</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="PB_02setRowCount">
          <property name="text">
           <string>设置行数</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QCheckBox" name="CB_04verticalVisible">
          <property name="text">
           <string>显示垂直表头</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="PB_07autoRowHeight">
          <property name="text">
           <string>自动调节行高</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="PB_01setHorizontalHeader">
          <property name="text">
           <string>设置水平表头</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="SB_rowCount">
          <property name="value">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="CB_02intervalRowBackground">
          <property name="text">
           <string>间隔行底色</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="PB_05appendRow">
          <property name="text">
           <string>添加行</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QPushButton" name="PB_09readContentsToTextEdit">
          <property name="text">
           <string>读取表格内容到文本框</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="PB_06deleteCurrentRow">
          <property name="text">
           <string>删除当前行</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QRadioButton" name="RB_01selectRow">
          <property name="text">
           <string>行选择</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QRadioButton" name="RB_02selectCell">
          <property name="text">
           <string>单元格选择</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitterV">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableWidget" name="TW_people">
        <property name="rowCount">
         <number>6</number>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>第一列</string>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>第二列</string>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>16</pointsize>
           <bold>false</bold>
          </font>
         </property>
        </column>
        <column/>
        <column/>
        <column/>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="2" column="2">
         <property name="text">
          <string/>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
        </item>
       </widget>
       <widget class="QPlainTextEdit" name="PTE_infomation"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PB_07autoRowHeight</sender>
   <signal>clicked()</signal>
   <receiver>TW_people</receiver>
   <slot>resizeRowsToContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PB_08autoColumnWidth</sender>
   <signal>clicked()</signal>
   <receiver>TW_people</receiver>
   <slot>resizeColumnsToContents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
